example/hidden.js

```diff
@@ -20,6 +22,6 @@ app.model(User);
 var apiPath = '/api';
 explorer(app, { basePath: apiPath });
 app.use(apiPath, loopback.rest());
-console.log('Explorer mounted at localhost:' + port + '/explorer');
+g.log('{{Explorer}} mounted at {{localhost:%s/explorer}}', port);
```

Comment:
:+1: 

@0candy, it was said somewhere that words with an uppercase letter to not get translated, does that include words at the beginning of a sentence?